From 2a964dabf86f7b1cdcc554b598d09b07ff52ce40 Mon Sep 17 00:00:00 2001
From: Simon Hausmann <simon.hausmann@slint-ui.com>
Date: Sun, 18 Jun 2023 11:35:07 +0200
Subject: [PATCH] WIP: Use a patched gettext to avoid cross-compiling gettext

 when building with Ycoto
---
 Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 1be45f0fe..a9e51ea3f 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -161,6 +161,9 @@ smol_str = { version = "0.3.1" }
 rayon = { version = "1.10.0", default-features = false }
 raw-window-handle-06 = { package = "raw-window-handle", version = "0.6", features = ["alloc"] }
 
+[patch.crates-io]
+gettext-sys = { git = "https://github.com/slint-ui/gettext-rs", branch = "simon/fix-linux-detection" }
+
 [profile.release]
 lto = true
 panic = "abort"
