From dc4ef40f5b6e4cb3b4af745a2ed8512c86715981 Mon Sep 17 00:00:00 2001
From: Simon Hausmann <simon.hausmann@slint-ui.com>
Date: Sun, 18 Jun 2023 11:35:07 +0200
Subject: [PATCH] WIP: Use a patched gettext to avoid cross-compiling gettext
 when building with Ycoto

---
 Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index cfb29e9f6..d2095364c 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -97,6 +97,9 @@ resvg = { version= "0.34.0", default-features = false, features = ["text"] }
 fontdb = { version = "0.14.1", default-features = false }
 send_wrapper = { version = "0.6.0" }
 
+[patch.crates-io]
+gettext-sys = { git = "https://github.com/slint-ui/gettext-rs", branch = "simon/fix-linux-detection" }
+
 [profile.release]
 lto = true
 panic = "abort"
