From e8b15fcdfa595d0587f62f2340c632ca6b032c58 Mon Sep 17 00:00:00 2001
From: Simon Hausmann <simon.hausmann@slint.dev>
Date: Sun, 18 Jun 2023 11:35:07 +0200
Subject: [PATCH] WIP: Use a patched gettext to avoid cross-compiling gettext
 when building with Yocto

Upstream-Status: Inappropriate [embedded specific]

---
 Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -213,6 +213,9 @@ objc2 = { version = "0.6.3" }
 # Not updated because of https://github.com/gluon-lang/lsp-types/issues/284
 lsp-types = "0.95.0"

+[patch.crates-io]
+gettext-sys = { git = "https://github.com/slint-ui/gettext-rs", branch = "simon/fix-linux-detection" }
+
 [profile.release]
 panic = "abort"

