From 0c3f95a28ed813e4a1d577a7ea9b77dc2500e48d Mon Sep 17 00:00:00 2001
From: Simon Hausmann <simon.hausmann@slint-ui.com>
Date: Sun, 18 Jun 2023 11:35:07 +0200
Subject: [PATCH] WIP: Use a patched gettext to avoid cross-compiling gettext
 when building with Ycoto

---
 Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 0b9d284dc..c19a739cd 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -128,6 +128,9 @@ send_wrapper = { version = "0.6.0" }
 cbindgen = { version = "0.26", default-features = false }
 toml_edit = "0.20"
 
+[patch.crates-io]
+gettext-sys = { git = "https://github.com/slint-ui/gettext-rs", branch = "simon/fix-linux-detection" }
+
 [profile.release]
 lto = true
 panic = "abort"
