From 4c7820a7297f150ee31901c1a5039341bef91779 Mon Sep 17 00:00:00 2001
From: Simon Hausmann <simon.hausmann@slint-ui.com>
Date: Sun, 18 Jun 2023 11:35:07 +0200
Subject: [PATCH] WIP: Use a patched gettext to avoid cross-compiling gettext
 when building with Ycoto

---
 Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index d91f69ae1..83430ef17 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -166,6 +166,9 @@ unicode-segmentation = { version = "1.12.0" }
 glow = { version = "0.16" }
 tikv-jemallocator = { version = "0.6" }
 
+[patch.crates-io]
+gettext-sys = { git = "https://github.com/slint-ui/gettext-rs", branch = "simon/fix-linux-detection" }
+
 [profile.release]
 lto = true
 panic = "abort"
