From 89b7eab0c00bccb52f2b5409b77405281bb76130 Mon Sep 17 00:00:00 2001
From: Simon Hausmann <simon.hausmann@slint-ui.com>
Date: Sun, 18 Jun 2023 11:35:07 +0200
Subject: [PATCH] WIP: Use a patched gettext to avoid cross-compiling gettext

 when building with Ycoto
---
 Cargo.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index a3f17d6f0..ad00224a7 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -185,6 +185,9 @@ skrifa = { version = "0.37" }
 windows = { version = "0.62" }
 windows-core = { version = "0.62.0" }
 
+[patch.crates-io]
+gettext-sys = { git = "https://github.com/slint-ui/gettext-rs", branch = "simon/fix-linux-detection" }
+
 [profile.release]
 lto = true
 panic = "abort"
